x-pack/filebeat/module/aws/elb/README.md

```diff
@@ -0,0 +1,64 @@
+Filebeat module for AWS ELB
+===
+
+Module for the AWS load balancers, it supports the following flavours:
+
+* ELB (Classic Load Balancer)
+* Application Load Balancer (V2 Load Balancer for HTTP)
+* Network Load Balancer (V2 Load Balancer for TCP and UDP - UDP not tested)
+
+Implementation based on the description of the access logs from the
+documentation that can be found in:
+
+* ELB: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html
+* Application LB: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+* Network LB: https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html
+
+Test files starting with `example` are copied or based on examples of this
+documentation.
+
+
+How to manual test this module
+===
+
+* Create an ELB and enable access logs for it. A terraform scenario is included
+  as example, read the section below about this if you want to use it.
+* Make some requests to the load balancer.
+* Configure filebeat, using the queue url from `terraform output sqs_queue_url`.
+```
+filebeat.modules:
+- module: aws
+  elb:
+   enabled: true
+   var.queue_url: <queue url>
+   var.credential_profile_name: <profile name>
+  s3access:
+   enabled: false
```

Comment:
I had an ALB setup for testing and realized if `s3access: enabled: false` is not set, then filebeat reports error. Is this normal for filebeat? If not, I can try look into it and create a separate PR to fix that.

Reply:
I am afraid that this is normal. Unlike in metricbeat, if you enable a module in filebeat, all its filesets are enabled by default. This may be an inconvenient in modules like this one where not all filesets are wanted. I remember discussing at some moment about changing this behaviour, but if we do it, we should do it in a major version, or as a configuration parameter disabled by default.

Reply:
Oh, I found it https://github.com/elastic/beats/issues/11943

Reply:
Yep thanks for the link!